common/names.go

```diff
@@ -87,5 +90,34 @@ func LocalVolumeKey(lv *localv1.LocalVolume) string {
 // GetProvisionedByValue is the the annotation that indicates which node a PV was originally provisioned on
 // the key is provCommon.AnnProvisionedBy ("pv.kubernetes.io/provisioned-by")
 func GetProvisionedByValue(node corev1.Node) string {
-	return fmt.Sprintf("local-volume-provisioner-%v-%v", node.Name, node.UID)
+	return fmt.Sprintf("local-volume-provisioner-%v", node.Name)
+}
+
+func PVMatchesProvisioner(pv corev1.PersistentVolume, provisionerName string) bool {
+	PVAnnotation, found := pv.Annotations[provCommon.AnnProvisionedBy]
+	if !found {
+		klog.InfoS("PV annotation not found - skipping.", "pvName", pv.GetName(), "provisionerName", provisionerName)
```

Comment:
Since this function `PVMatchesProvisioner` is called by several Reconcile loops, I think it's a bad idea to call `InfoS` in each case... this seems like it would flood the logs over time.

Reply:
Ack - the logs seemed already quite flooded but good point, this does not have to be info level. We could increase the log level (see current version of the patch). That should be safe because I don't see -v flag being used for the deployment and default should be lower than 4.

That's what I initially wanted to do. However, I noticed that diskmaker (binary) does not seem to support changing log levels atm. That's probably why it's not in the deployment/daemonset. Maybe I'm missing something?

Or we can just remove these messages.

Reply:
Thanks, I think setting these to log level 4 makes sense.
The reason logLevel is not propagated to diskmaker is due to https://bugzilla.redhat.com/show_bug.cgi?id=2058001 -- I haven't had time to work on it, feel free to steal that bug if you get bored :)